docs/source/user-guide/vision/quickstarts/plot_detection_tutorial.py

```diff
@@ -138,49 +123,8 @@ def __len__(self):
 train_loader = DataLoader(train_set, batch_size=64, collate_fn=(lambda batch: tuple(zip(*batch))))
```

Comment:
I was wondering about that. Because originally after this, there was some visualization, and printing the batch. 
I removed the visualization (and put it later because it uses the VisionData), but then the problem is that we have A LOT of code without anything running. So I kept the "print the batch" thing, which requires the dataloader...

I'm not sure which answer is the correct one though.

Reply:
hmm i see your point.
Maybe as a compromise, you can print stuff based on the dataset object - you can print the len of dataset, shape of image at index 0 and the label for index 0